azurelinuxagent/common/event.py

```diff
@@ -111,6 +111,7 @@ class WALAEventOperation:
     OpenSsl = "OpenSsl"
     Partition = "Partition"
     PersistFirewallRules = "PersistFirewallRules"
+    Policy = "Policy"
```

Comment:
Maybe 'ExtensionsPolicy' to be clear this flag is for extensions only

Reply:
Eventually, this label would be used for all in-guest policy logs, do we want to create a separate label for each policy type (extensions policy, goal state policy, etc)? I left it as "policy" to keep it simpler in logs.